### ISSUE TITLE
CR Failed to download m3u8 information: HTTP Error 403: Forbidden

### ISSUE THREAD

**Issue by einst3** [yt-dlp/yt-dlp#9497]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

united states/eastern

### Provide a description that is worded well enough to be understood

i cant download anything from crunchyrole titles but seems there is some issue with cloudflare i think. here is list-subs and list-formats. also included download at end. first time posting please be gentle with me =)

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [x] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[Enter the video URL of the video to download] https://www.crunchyroll.com/watch/GPWUK1335/we-dont-need-aces
[debug] Command-line config: ['--config-locations', '__config1c1.txt', 'https://www.crunchyroll.com/watch/GPWUK1335/we-dont-need-aces']
[debug] | Config "__config1c1.txt": ['--ffmpeg-location', 'ffmpeg', '--verbose', '--no-check-certificate', '--no-geo-bypass', '--write-subs', '--sub-langs', 'en.*', '--trim-filenames', '150', '--sub-format', 'ass/srt/best', '-N', '5', '--limit-rate', '3.2M', '-f', 'bv[height<1080][height>=360]+ba/b[height<1080][height>=360]', '-P', 'downloads', '-o', '%(series)s_S0%(season_number)sE0%(episode_number)s_%(episode)s.%(ext)s', '--cookies-from-browser', 'chrome', '--user-agent', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36', '--remux-video', 'mkv', '--embed-subs', '--ppa', 'EmbedSubtitle:-disposition:s:0 default', '--embed-metadata', '--parse-metadata', ':(?P<meta_purl>)', '--parse-metadata', ':(?P<meta_comment>)', '--parse-metadata', ':(?P<meta_description>)', '--parse-metadata', ':(?P<meta_title>)', '--parse-metadata', ':(?P<meta_episode_id>)', '--parse-metadata', ':(?P<meta_date>)', '--parse-metadata', ':(?P<meta_season_number>)', '--parse-metadata', ':(?P<meta_episode_sort>)', '--no-embed-info-json', '--no-write-comments', '--no-embed-chapters']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2024.03.10 from yt-dlp/yt-dlp [615a84447] (win32_dir)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg N-114254-g030f4925b8-20240318 (setts), ffprobe N-114254-g030f4925b8-20240318
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.1.0, certifi-2024.02.02, mutagen-1.47.0, requests-2.31.0, sqlite3-3.35.5, urllib3-2.2.1, websockets-12.0
[debug] Proxy map: {}
Extracting cookies from chrome
[debug] Extracting cookies from: "C:\Users\user\AppData\Local\Google\Chrome\User Data\Default\Network\Cookies"
[debug] Found local state file at "C:\Users\user\AppData\Local\Google\Chrome\User Data\Local State"
Extracted 51 cookies from chrome
[debug] cookie version breakdown: {'v10': 51, 'other': 0, 'unencrypted': 0}
[debug] Request Handlers: urllib, requests, websockets
[debug] Loaded 1803 extractors
[crunchyroll] Extracting URL: https://www.crunchyroll.com/watch/GPWUK1335/we-dont-need-aces
[debug] [crunchyroll] Using cxApiParam=noaihdevm_6iyg0a8l0q
[crunchyroll] Authenticating with grant_type=etp_rt_cookie
[crunchyroll] GPWUK1335: Downloading object info JSON (cms)
[crunchyroll] Retrieving signed policy
[crunchyroll] GPWUK1335: Downloading stream info JSON (signed cms)
[crunchyroll] To get all formats of a hardsub language, use "--extractor-args crunchyrollbeta:hardsub=<language_code or all>". See https://github.com/yt-dlp/yt-dlp#crunchyrollbeta-crunchyroll for more info
[crunchyroll] GPWUK1335: Downloading adaptive_hls HLS manifest
WARNING: [crunchyroll] Failed to download m3u8 information: HTTP Error 403: Forbidden
[crunchyroll] GPWUK1335: Downloading chapter info
[info] GPWUK1335: Downloading subtitles: en-US
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
[debug] Searching for '(?P<meta_purl>)' in ''
[MetadataParser] Parsed meta_purl from '': ''
[debug] Searching for '(?P<meta_comment>)' in ''
[MetadataParser] Parsed meta_comment from '': ''
[debug] Searching for '(?P<meta_description>)' in ''
[MetadataParser] Parsed meta_description from '': ''
[debug] Searching for '(?P<meta_title>)' in ''
[MetadataParser] Parsed meta_title from '': ''
[debug] Searching for '(?P<meta_episode_id>)' in ''
[MetadataParser] Parsed meta_episode_id from '': ''
[debug] Searching for '(?P<meta_date>)' in ''
[MetadataParser] Parsed meta_date from '': ''
[debug] Searching for '(?P<meta_season_number>)' in ''
[MetadataParser] Parsed meta_season_number from '': ''
[debug] Searching for '(?P<meta_episode_sort>)' in ''
[MetadataParser] Parsed meta_episode_sort from '': ''
ERROR: [crunchyroll] GPWUK1335: Requested format is not available. Use --list-formats for a list of available formats
Traceback (most recent call last):
  File "yt_dlp\YoutubeDL.py", line 1594, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1750, in __extract_info
  File "yt_dlp\YoutubeDL.py", line 1809, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 2930, in process_video_result
yt_dlp.utils.ExtractorError: [crunchyroll] GPWUK1335: Requested format is not available. Use --list-formats for a list of available formats


[debug] Command-line config: ['--config-locations', '__config1c1.txt', 'https://www.crunchyroll.com/watch/GYGGPN7DY/its-been-six-years']
[debug] | Config "__config1c1.txt": ['--ffmpeg-location', 'ffmpeg', '--verbose', '--no-check-certificate', '--no-geo-bypass', '--write-subs', '--sub-langs', 'en.*', '--trim-filenames', '150', '--sub-format', 'ass', '-N', '5', '--limit-rate', '0.98M', '-f', 'bv[height<1080][height>=360]+ba/b[height<1080][height>=360]', '--extractor-args', 'crunchyrollbeta:hardsub=all,None', '-P', 'downloads', '-o', '%(series)s_S0%(season_number)sE0%(episode_number)s_%(episode)s.%(ext)s', '--cookies-from-browser', 'chrome', '--user-agent', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36', '--remux-video', 'mkv', '--embed-subs', '--ppa', 'EmbedSubtitle:-disposition:s:0 default', '--embed-metadata', '--parse-metadata', ':(?P<meta_purl>)', '--parse-metadata', ':(?P<meta_comment>)', '--parse-metadata', ':(?P<meta_description>)', '--parse-metadata', ':(?P<meta_title>)', '--parse-metadata', ':(?P<meta_episode_id>)', '--parse-metadata', ':(?P<meta_date>)', '--parse-metadata', ':(?P<meta_season_number>)', '--parse-metadata', ':(?P<meta_episode_sort>)', '--no-embed-info-json', '--no-write-comments', '--no-embed-chapters']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2024.03.10 from yt-dlp/yt-dlp [615a84447] (win32_dir)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg N-114254-g030f4925b8-20240318 (setts), ffprobe N-114254-g030f4925b8-20240318
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.1.0, certifi-2024.02.02, mutagen-1.47.0, requests-2.31.0, sqlite3-3.35.5, urllib3-2.2.1, websockets-12.0
[debug] Proxy map: {}
Extracting cookies from chrome
[debug] Extracting cookies from: "C:\Users\einst\AppData\Local\Google\Chrome\User Data\Default\Network\Cookies"
[debug] Found local state file at "C:\Users\einst\AppData\Local\Google\Chrome\User Data\Local State"
Extracted 51 cookies from chrome
[debug] cookie version breakdown: {'v10': 51, 'other': 0, 'unencrypted': 0}
[debug] Request Handlers: urllib, requests, websockets
[debug] Loaded 1803 extractors
[crunchyroll] Extracting URL: https://www.crunchyroll.com/watch/GYGGPN7DY/its-been-six-years
[debug] [crunchyroll] Using cxApiParam=noaihdevm_6iyg0a8l0q
[crunchyroll] Authenticating with grant_type=etp_rt_cookie
ERROR: [crunchyroll] GYGGPN7DY: Request blocked by Cloudflare; navigate to Crunchyroll in your browser, then pass the fresh cookies (with --cookies-from-browser or --cookies) and your browser's User-Agent (with --user-agent)
  File "yt_dlp\extractor\common.py", line 732, in extract
  File "yt_dlp\extractor\crunchyroll.py", line 333, in _real_extract
  File "yt_dlp\extractor\crunchyroll.py", line 127, in _call_api
  File "yt_dlp\extractor\crunchyroll.py", line 108, in _call_base_api
  File "yt_dlp\extractor\crunchyroll.py", line 94, in _update_auth


```

**Comment by bashonly** [comment]
Duplicate of #9439

But also see #9453
